bin/cython-generate-lexicon.py

```diff
@@ -63,12 +66,12 @@ def main():
         print("Warning: generated code section not found - code not inserted")
         return
 
-    if sys.argv[1] == "--here":
+    if arg == "--here":
         outfile = "Lexicon.py"
-    elif sys.argv[1] == "--overwrite":
+    elif arg == "--overwrite":
         outfile = Lexicon.__file__
     else:
-        raise ValueError('argv: "{0}" not recognised', sys.argv[1])
+        raise ValueError('argument "{0}" not recognised', arg)
```

Comment:
I think this should say `'…'.format(arg)`, but since this case is already handled above, a bare `else:` for the second case, maybe with an `assert`, seems enough.